docs/user/security-log.rst

```diff
@@ -10,49 +10,43 @@ so that you can confirm this access was from the intended person.
 Security logs are only visible to organization owners.
 You can invite other team members as owners.
 
+.. seealso::
+
+   :doc:`/legal/security-policy`
+     General information and reference about how security is handled on Read the Docs.
+
 User security log
 -----------------
 
-We store user security logs for the last 90 days, and track the following events:
+We store a user security log for the latest 90 days of activity.
+This log is useful to validate that no unauthorized logins have occurred on your user account.
+
+The security log tracks the following events:
 
 - Authentication on the dashboard
 - Authentication on documentation pages (:doc:`/commercial/index` only)
 
 Authentication failures and successes are both tracked.
 
-To access your logs:
-
-- Click on :guilabel:`<Username dropdown>`
-- Click on :guilabel:`Settings`
-- Click on :guilabel:`Security Log`
-
-This log is useful to validate that no unauthorized logins have occured on your user account.
+Logs are available in :menuselection:`<Username dropdown> --> Settings --> Security Log`.
 
 Organization security log
 -------------------------
 
-.. note::
-
-   This feature exists only on :doc:`/commercial/index`.
+.. include:: /shared/admonition-rtd-business.rst
 
 The length of log storage varies with your plan,
 check our `pricing page <https://readthedocs.com/pricing/>`__ for more details.
-We track the following events:
+Your organization security log is a great place to check periodically to ensure there hasn't been unauthorized access to your organization.
+
+Organization logs track the following events:
 
 - Authentication on documentation pages from your organization
 - User access to every documentation page from your organization (**Enterprise plans only**)
```

Comment:
We are also tracking downloads, invitations sent/revoked/accepted.

Reply:
Added `User access to download documentation pages  (**Enterprise plans only**)` - supposing this is an enterprise feature or?

Reply:
yeah, this is basically a subset of "page views"

https://github.com/readthedocs/readthedocs.org/blob/ce1a81a2f8bd4644e52571ecf5663cff3a6c1c4c/readthedocs/proxito/views/mixins.py#L195-L200